test_utils/integration_test_translate_wav.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+# -*- mode:sh -*-
+
+setup() {
+    cd tools || exit 1
+    make moses.done
+    cd - || exit 1
+
+    tmpdir=/tmp/espnet-test-st
+}
+
+
+@test "translate_wav" {
+    cd ./egs/fisher_callhome_spanish/st1/
+    wav=../../../test_utils/st_test.wav
+
+    # TODO(karita): why tar results in error?
```

Comment:
You mean the error in google drive download?
The download process is changed depending on the data size.

Reply:
Hmm, the error in untar seems to disappear. Maybe my previous bash (set -euo?) option was wrong.